Kernel/Net/NetworkTask.cpp

```diff
@@ -329,6 +330,38 @@ void flush_delayed_tcp_acks()
     }
 }
 
+void send_tcp_rst(const IPv4Packet& ipv4_packet, const TCPPacket& tcp_packet, RefPtr<NetworkAdapter> adapter)
```

Comment:
```suggestion
void send_tcp_rst(IPv4Packet const& ipv4_packet, TCPPacket const& tcp_packet, RefPtr<NetworkAdapter> adapter)
```

This should be east const. Same for the other code :)

Reply:
Fixed :) 
I'll open up a separate PR to update the rest of the functions.